pnpm-lock.yaml

```diff
@@ -4,34 +4,32 @@ specifiers:
   '@algolia/client-search': ^4.15.0
```

Comment:
@alexiglesias93 do we need this for our usecase?

Reply:
nope, please uninstall it

Reply:
removed: https://github.com/finsweet/ts-utils/pull/24/commits/a435a84e0f25283b1282f50904f7e0cc95079ac8